apps/public/src/app/(general)/contact/actions.ts

```diff
@@ -0,0 +1,34 @@
+'use server'
+
+import { postMeldingByMeldingIdContact } from '@meldingen/api-client'
+import { cookies } from 'next/headers'
+import { redirect } from 'next/navigation'
+
+export const postContactForm = async (_: unknown, formData: FormData) => {
+  const cookieStore = await cookies()
+
+  const meldingId = cookieStore.get('id')
+  const token = cookieStore.get('token')
+
+  if (!meldingId || !token) return undefined
+
+  const email = formData.get('email')
+  const phone = formData.get('phone')
+
+  if (email || phone) {
+    try {
+      postMeldingByMeldingIdContact({
+        meldingId: Number(meldingId.value),
```

Comment:
Should we use `Number` everywhere for the `meldingId` conversion?

I like that it doesn't convert '10px' to 10 or anything. However, it coverts null and '' to 0, which might be a bigger issue.

Reply:
You mean replacing parseInt() at some places?

Reply:
Yeah, we should do the same thing everywhere we convert `meldingId` from string to number. I think I prefer `parseInt` though, because of the whole `null` = 0 thing.

Reply:
gree!